### PR TITLE
DF-2426 | Fix Null or None meta being passed in a plugin

### PR DESCRIPTION
**Pull request by lmilby-r7**
When running a test if `meta` is passed as None, you'll get the message below
```
Traceback (most recent call last):
  File "/usr/local/bin/icon_jq", line 4, in <module>
    __import__('pkg_resources').run_script('jq-rapid7-plugin==2.0.4', 'icon_jq')
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 666, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1453, in run_script
    exec(script_code, namespace, namespace)
  File "/usr/local/lib/python3.7/site-packages/jq_rapid7_plugin-2.0.4-py3.7.egg/EGG-INFO/scripts/icon_jq", line 32, in <module>
  File "/usr/local/lib/python3.7/site-packages/jq_rapid7_plugin-2.0.4-py3.7.egg/EGG-INFO/scripts/icon_jq", line 28, in main
  File "/usr/local/lib/python3.7/site-packages/komand-1.0.1-py3.7.egg/komand/cli.py", line 188, in run
    args.func(args)
  File "/usr/local/lib/python3.7/site-packages/komand-1.0.1-py3.7.egg/komand/cli.py", line 138, in run_step
    return self.execute_step(is_test=False, is_debug=args.debug)
  File "/usr/local/lib/python3.7/site-packages/komand-1.0.1-py3.7.egg/komand/cli.py", line 126, in execute_step
    output = self.plugin.handle_step(msg, is_test=is_test, is_debug=is_debug)
  File "/usr/local/lib/python3.7/site-packages/komand-1.0.1-py3.7.egg/komand/plugin.py", line 283, in handle_step
    self.connection.meta.set_workflow(input_message['body'].get('meta', None))
  File "/usr/local/lib/python3.7/site-packages/komand-1.0.1-py3.7.egg/komand/plugin.py", line 139, in set_workflow
    if input_message.get("workflow_uid"):
AttributeError: 'NoneType' object has no attribute 'get'
```

To fix this we added a checking in handle_step in the plugin.py to set it to an empty dict.

Before: 
Screen Shot 2019-03-26 at 5 24 16 PM

After:
Screen Shot 2019-03-25 at 3 12 17 PM

### CODE REVIEW

File: komand/plugin.py [hunk 1]
```diff
@@ -280,7 +280,11 @@ def handle_step(self, input_message, is_test=False, is_debug=False):
         :param is_debug:
         :return:
         """
-        self.connection.meta.set_workflow(input_message['body'].get('meta', None))
+        input_message_meta = input_message["body"].get("meta", None)
```
Comment (mrinehart-r7) [inline komand/plugin.py]: No need to default it to `None`, `get()` automatically does that.

Comment (mrinehart-r7) [reply komand/plugin.py]:
```suggestion
        input_message_meta = input_message["body"].get("meta", {})
```

File: komand/plugin.py [hunk 2]
```diff
@@ -280,7 +280,11 @@ def handle_step(self, input_message, is_test=False, is_debug=False):
...
+
+        if input_message_meta == None:
```
Comment (mrinehart-r7) [inline komand/plugin.py]: Use the code suggestion above and drop lines 285, 286

Comment (lmilby-r7) [reply komand/plugin.py]: I know its odd, but since `meta` exists it will pull that value
Screen Shot 2019-03-26 at 5 49 00 PM

Comment (mrinehart-r7) [reply komand/plugin.py]: Wait, weird! So The "meta" key DOES exist, but there's nothing in it so it it becomes None? X_X

Comment (lmilby-r7) [reply komand/plugin.py]: I passed None just incase some weird stuff happens.

**Pull request merged**